src/utils/bugzilla/chartJs/burndownDataFormatter.js

```diff
@@ -0,0 +1,52 @@
+import generateLineChartStyles from '../../chartJs/generateLineChartStyles';
+import SETTINGS from '../../../settings';
+
+const bugsByCreationDate = (bugs, startDate) => {
+    // Count bugs created on each day
+    const byCreationDate = bugs.reduce((result, { creation_time }) => {
+        const date = creation_time.substring(0, 10);
+        // startDate enables counting into a starting date all previous data points
+        const actualDate = (startDate && date < startDate) ? startDate : date;
```

Comment:
Could you clarify how this logic works: `startDate && date < startDate`?

Reply:
I can understand why the parenthesis makes it easier for our brains to understand what is going on. I will add it.